randovania/games/prime2/exporter/patch_data_factory.py

```diff
@@ -650,30 +672,58 @@ def add_dock_type_changes(self, regions_patch_data: dict):
             dock_patch_data = self._get_dock_patch_data(regions_patch_data, dock)
             dock_patch_data.update(changes)
 
-    def new_patcher_configuration(self):
-        worlds_patch_data = {
-            "Temple Grounds": {
-                "areas": {
-                    "Dynamo Chamber": {
-                        "layers": {
-                            "1st Pass Scripting": False,
-                            "2nd Pass Scripting": True,
-                        }
-                    },
-                    "Trooper Security Station": {
-                        "layers": {
-                            "1st Pass": False,
-                            "2nd Pass": True,
-                        }
-                    },
-                }
-            }
+    def add_new_patcher_elevators(self, regions_patch_data: dict):
+        for node, connection in self.patches.all_elevator_connections():
+            region, area = self._add_area_to_regions_patch(regions_patch_data, node)
+            area_patches = regions_patch_data[region.name]["areas"][area.name]
+            area_patches["elevators"].append({
+                "instance_id": node.extra["teleporter_instance_id"],
+                "target_assets": _area_identifier_to_json(
+                    self.game.region_list,
+                    connection
+                ),
+                "target_strg": node.extra["scan_asset_id"],
```

Comment:
Not every teleporter has one of these. Do the others have null in the extra?

Reply:
they do indeed have null in their extra